paper-autocomplete-suggestions.html

```diff
@@ -176,756 +159,764 @@
       </template>
 
       <!-- Custom template -->
-      <content id="templates" select="[autocomplete-custom-template]"></content>
+      <slot id="templates" name="autocomplete-custom-template"></slot>
     </div>
   </template>
-</dom-module>
 
-<script>
-  (function () {
+  <script>
```

Comment:
It seems the whole script tag has been formatted because it appears that all lines has changed. Can you revert this formatting? Thanks